sdk/storage/Azure.Storage.Blobs/tests/SessionRecords/BlobBaseClientTests/OpenReadAsync_Seek_Position(10,Current).json

```diff
@@ -103,27 +108,30 @@
         "x-ms-access-tier-inferred": "true",
         "x-ms-blob-type": "BlockBlob",
         "x-ms-client-request-id": "669ff921-aacd-4a40-f5de-1c7597e9d84f",
-        "x-ms-creation-time": "Wed, 07 Oct 2020 19:40:06 GMT",
-        "x-ms-last-access-time": "Wed, 07 Oct 2020 19:40:06 GMT",
+        "x-ms-creation-time": "Wed, 17 Feb 2021 18:45:10 GMT",
+        "x-ms-is-current-version": "true",
+        "x-ms-last-access-time": "Wed, 17 Feb 2021 18:45:10 GMT",
         "x-ms-lease-state": "available",
         "x-ms-lease-status": "unlocked",
-        "x-ms-request-id": "19fd1804-501e-0064-3be1-9c2b5a000000",
+        "x-ms-request-id": "84fe38cc-101e-005a-155d-05bc25000000",
         "x-ms-server-encrypted": "true",
-        "x-ms-version": "2020-06-12"
+        "x-ms-version": "2020-06-12",
+        "x-ms-version-id": "2021-02-17T18:45:10.7244038Z"
       },
       "ResponseBody": []
     },
     {
       "RequestUri": "https://seanmcccanary3.blob.core.windows.net/test-container-52bce973-743d-f688-387a-f42ef0455460/test-blob-436b4032-bbb1-b1a9-b087-a6dcfe9afef2",
       "RequestMethod": "GET",
       "RequestHeaders": {
+        "Accept": "application/xml",
```

Comment:
This doesn't make sense here also. we don't know what's the content of blob and also response content type comes back different than "accept" indicates.
Maybe we should just get rid of this for all storage calls, it doesn't seem to do anything useful (but adds more bytes to be pushed over the wire).

Reply:
Same as above.